fastly/resource_fastly_service_wasm_v1.go

```diff
@@ -0,0 +1,43 @@
+package fastly
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+// Ordering is important - stored is processing order
+// Conditions need to be updated first, as they can be referenced by other
+// configuration objects (Backends, Request Headers, etc)
+
+var wasmService = &BaseServiceDefinition{
+	Type: "wasm",
+	Attributes: []ServiceAttributeDefinition{
+		NewServiceDomain(),
+		NewServiceHealthCheck(),
+		NewServiceBackend(),
+		//These have been commented out until they are satisfactorily integrated with Wasm resources
```

Comment:
Opinion: personally, I would prefer removing these as a comment for now and adding them here once we integrate logging for WASM services.